usb/esp_tinyusb/include/tusb_msc_storage.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stddef.h>
+#include "esp_err.h"
+
+#if SOC_SDMMC_HOST_SUPPORTED
+#include "driver/sdmmc_host.h"
+#endif
+
+
+#if SOC_SDMMC_HOST_SUPPORTED
+/**
+ * @brief Configuration structure for sdmmc initialization
+ *
+ * User configurable parameters that are used while
+ * initializing the sdmmc media.
+ */
+typedef struct {
+    sdmmc_host_t host;
+    sdmmc_slot_config_t slot_config;
+} tinyusb_msc_sdmmc_config_t;
+#endif
+
+/**
+ * @brief Configuration structure for spiflash initialization
+ *
+ * User configurable parameters that are used while
+ * initializing the SPI Flash media.
+ */
+typedef struct {
+    // Place holder. Currently, nothing to configure
+} tinyusb_msc_spiflash_config_t;
+
+/**
+ * @brief Configuration structure for storage initialization
+ */
+typedef struct {
+    union {
+        tinyusb_msc_spiflash_config_t *spiflash_config;
+#if SOC_SDMMC_HOST_SUPPORTED
+        tinyusb_msc_sdmmc_config_t *sdmmc_config;
```

Comment:
Could these be values instead of pointers?

Reply:
done